{"name": "__main__", "filename": "time_logging.py", "lineno": 104, "process": 19496, "message": "A debug message", "severity": "DEBUG", "timestamp": "2024-01-02T23:36:37Z"}
{"name": "__main__", "filename": "time_logging.py", "lineno": 105, "process": 19496, "message": "An error message", "severity": "ERROR", "timestamp": "2024-01-02T23:36:37Z"}
{"name": "__main__", "filename": "logging_demo.py", "lineno": 73, "process": 12920, "message": "A debug message", "severity": "DEBUG", "timestamp": "2024-01-03T09:48:21Z"}
{"name": "__main__", "filename": "logging_demo.py", "lineno": 74, "process": 12920, "message": "An error message", "severity": "ERROR", "timestamp": "2024-01-03T09:48:21Z"}
